### PR TITLE
Added Tests for saving and loading JSON objects

### PR DESCRIPTION
**Pull request by ThePhoneBeLinging**

### CODE REVIEW

**Review by emilsavino — APPROVED** [review]

File: src/test/java/JSONParserTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    {
+        workouts = new ArrayList<>();
+
+        Workout workout = new Workout();
+        for (Exercise exercise : exercises)
+        {
+            workout.addExercise(exercise);
+        }
+        workouts.add(workout);
+        workout = new Workout();
```
Comment (WettyWednesday) [inline src/test/java/JSONParserTest.java:39]: Is there a reason you are creating a new instance after populating the workout object?

File: src/test/java/JSONParserTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    private String exercisesFilePath;
+    private String workoutsFilePath;
+    private List<Exercise> exercises;
+    private List<Workout> workouts;
+
+    private void initExercises()
+    {
+        exercises = new ArrayList<>();
+
+        Exercise pushup = new Exercise("Pushup", "Push your body up and down", "Strength", "Chest", "None", "Easy", "None");
```
Comment (WettyWednesday) [inline src/test/java/JSONParserTest.java:23]: Can the JSONParser handle mutliple bodyparts?

**Review by WettyWednesday — CHANGES_REQUESTED** [review]

File: src/test/java/JSONParserTest.java (hunk 1 above)
Comment (ThePhoneBeLinging) [reply src/test/java/JSONParserTest.java:39]: Because it is a pointer

File: src/test/java/JSONParserTest.java (hunk 2 above)
Comment (ThePhoneBeLinging) [reply src/test/java/JSONParserTest.java:23]: Yes

**Review by WettyWednesday — APPROVED** [review]

**Pull request merged**